app/src/main/java/org/oppia/android/app/settings/profile/ProfileEditFragmentPresenter.kt

```diff
@@ -26,17 +26,16 @@ class ProfileEditFragmentPresenter @Inject constructor(
   private val activity: AppCompatActivity,
   private val fragment: Fragment,
   private val oppiaLogger: OppiaLogger,
+  private val profileEditViewModel: ProfileEditViewModel,
   private val profileManagementController: ProfileManagementController
 ) {
 
-  @Inject
-  lateinit var profileEditViewModel: ProfileEditViewModel
```

Comment:
Why is this change required?

Reply:
I thought to include the dependency at compile time rather than runtime shall I revert this?

Reply:
Yes please revert this here. If you think compile time is better suggest doing it in separate PR with explanation.